invokeai/app/api/routers/models.py

```diff
@@ -315,20 +315,21 @@ async def list_ckpt_configs(
     return ApiDependencies.invoker.services.model_manager.list_checkpoint_configs()
     
         
-@models_router.get(
+@models_router.post(
     "/sync",
     operation_id="sync_to_config",
     responses={
         201: { "description": "synchronization successful" },
     },
     status_code = 201,
-    response_model = None
+    response_model = bool
 )
 async def sync_to_config(
 )->None:
```

Comment:
let's make this match the `response_model` & return type

```suggestion
)->bool:
```

Reply:
Whoopsies. Missed it.